docs/decisions/0007-adjustments.rst

```diff
@@ -0,0 +1,40 @@
+0007 Ledger Adjustments
+#######################
+
+Status
+******
+**Accepted** (October 2023)
+
+Context
+*******
+We'd like to have the abilitiy to manually adjust the balance of ledgers
+in a somewhat unstructured way - that is, in a way that doesn't reverse
+an existing transaction.
+
+Decision
+********
+We'll introduce an ``Adjustment`` model that inherits from the abstract
+``BaseTransaction`` model:
+
+* This is consistent with our existing pattern of modeling any modification
+  to a ledger via models that inherit from ``BaseTransaction`` (like we
+  do with ``Reversal``).
+* Creating an ``Adjustment`` will cause a new ``Transaction`` to be written
+  to the adjustment's ledger object, with the same quantity as the adjustment record.
```

Comment:
This doesn't smell good for the simplicity of being able to sum() the quantities of all objects related to a ledger.  Now if I do sum(all transactions, all reversals, all adjustments), the result is a double-count of adjustments.  So I have to just know not to include adjustments in the sum, which is unintuitive because it extends from BaseTransaction so it seems summable.

Is it not possible to just make adjustments not FK to a transaction?  They extend from BaseTransaction so already FK to the ledger.

Reply:
If we want to retain support for the outcome of reversing an adjustment, I feel like that could be achievable with a negative value adjustment that just isn't self-referential.

Reply:
Yea, I'm also okay with (3).  Not inheriting a quantity field from BaseTransaction clears up any possible confusion IMO.

Aside from "why don't Reversals operate this way, too", as you mentioned, I also wonder "why don't initial transactions operate this way too"?  I guess initial transactions don't have any additional metadata so there's nothing to note and no fields to add to the note model.